docs/tutorial.rst

```diff
@@ -1,4 +1,209 @@
 Tutorial
 ========
 
-*Coming soon!*
+In this document, we are going to develop an application that will
+convert a given time (and date) between two time zones.
+The user will enter a date and a time,
+and select two time zones from a list of all available time zones,
+one for the source location and one for the destination location.
+
+.. note::
+
+   Time zones are represented in "region/location" format,
+   as in "Australia/Tasmania".
+   While most such codes have two components,
+   some contain only one (like "UTC"),
+   and some contain more than two (like "America/North_Dakota/New_Salem".
+   Also note that spaces in region and location names are replaced
+   with underscores.
+   See the `Wikipedia list`_ for more information.
+
+
+Prerequisites
+-------------
+
+First of all, we have to install the dependencies.
+These are: *clastic* (obviously) and *dateutil*.
+Note that the PyPI package name for *dateutil* is *python-dateutil*::
+
+  pip install clastic python-dateutil
+
+
```

Comment:
Probably worth mentioning that any version of Python will work. And it might be useful to link to something about basic development setup with virtualenv. Maybe [this](https://full-stack-development.readthedocs.io/en/latest/team_project_workflow/virtualenv.html) or [this](https://www.fullstackpython.com/virtual-environments-virtualenvs-venvs.html)?

Reply:
"Any version of Python" sounds too broad to me. Do the dependencies all support all versions?

Regarding virtual environments, the first link is for virtualenv (therefore Python 2), and te second one doesn't directly contain any information, but further links. How about the official documentation?

https://docs.python.org/3/tutorial/venv.html

Reply:
Yeah, that's pretty broad. At the time of writing it's tested against 2.7, 3.5-3.7, and PyPy2.

I'm not too picky about the virtualenv/venv link, it's mostly there to acknowledge that just running `pip install` on its own will fail in most cases. Better to hint at something other than `sudo pip install`. The official docs are fine.